test/fixtures/output/schema-registry/region/list-all.golden

```diff
@@ -1,4 +1,4 @@
-    ID    | Cloud | Region Name |    Display Name    |   Packages    
-----------+-------+-------------+--------------------+---------------
-  sgreg-1 | aws   | us-east-2   | Ohio (us-east-2)   | [advanced]    
-  sgreg-2 | gcp   | us-central1 | Iowa (us-central1) | [essentials]  
+    ID    |        Name        | Cloud | Region Name |  Packages   
```

Comment:
Did you forget to commit your test file?

Reply:
It's there - `schema_registry_test.go` 
https://github.com/confluentinc/cli/pull/1626/files#diff-768a296a821baed8fa8db2e2e6f1753a4fb66ffe71a3dd0f14f4f2dab573c8b0

Reply:
weird, it wasn't showing up for me.